lib/common/include/scipp/common/numeric.h

```diff
@@ -98,21 +98,26 @@ constexpr auto pow(const B base, const E exponent) noexcept {
   }
 }
 
-template <class T> bool isnan([[maybe_unused]] T x) {
+template <class T> bool isnan([[maybe_unused]] const T &x) {
```

Comment:
I am not happy that I had to change this to `const &`, but otherwise I'd have to pass also the large types (for now: `Eigen::Vector3d`) by value. For some `Eigen` types this is even forbidden completely. Is there a better way?

Reply:
I don't think this causes any harm. This function will probably be optimized away anyway.